README.md

```diff
@@ -111,6 +111,14 @@ python main.py \
     --device cuda:0
 ```
 
+GPTQ models can be loaded by specifying their file names in `,quantized=NAME` in the `model_args` argument:
```

Comment:
It might be a good idea to call them "GPTQ _quantized_ models" to emphasize to an unfamiliar user what the usecase is for.